tests/test_inter_area_agent.py

```diff
@@ -120,16 +120,14 @@ def delete_offer(self, *args):
     def delete_bid(self, *args):
         pass
 
-    def offer(self, price, energy, seller, iaa_fee: bool = False):
+    def offer(self, price, energy, seller):
         self.offer_count += 1
-        if iaa_fee:
-            price = price * (1 + self.transfer_fee_ratio) + self.transfer_fee_const * energy
+        price = price * (1 + self.transfer_fee_ratio) + self.transfer_fee_const * energy
         self.forwarded_offer = Offer(self.forwarded_offer_id, price, energy, seller, market=self)
         return self.forwarded_offer
 
-    def bid(self, price, energy, buyer, seller, iaa_fee: bool = False):
-        if iaa_fee:
-            price = price * (1 - self.transfer_fee_ratio) + self.transfer_fee_const * energy
+    def bid(self, price, energy, buyer, seller):
+        price = price * (1 - self.transfer_fee_ratio) + self.transfer_fee_const * energy
```

Comment:
```suggestion
        price = price * (1 - self.transfer_fee_ratio) - self.transfer_fee_const * energy
```

Reply:
The motivation to me for this change is if transfer_fee_ratio is reducing the bid_price, so should transfer_fee_const. Let me know if it makes sense now?